web-app/src/app/stitch-patterns.html

```diff
@@ -73,14 +73,30 @@
           patternInfos: {
             type: Array,
             value: []
-          }
+          },
+          active: {
+              type: Boolean,
+              reflectToAttribute: true,
+              observer: 'activeChanged'
+            }
         }
       }
 
-      constructor() {
-        super();
-        const json = localStorage.getItem("patternInfos");
-        if (json) this.patternInfos = JSON.parse(json);
+      activeChanged() {
+          if (!this.active) {
+            this.patternInfos = [];
+          }
+          else {
+            this.uploadPatterns();
+          }
+      }
+
+      async uploadPatterns() {
```

Comment:
"Upload" mean that user uploads something to the server. In this case name should be something like `getPatterns`. And there is no need to save them to localstorage anymore for now.